pegasus/sites.v3/advocacy.code.org/public/index.haml

```diff
@@ -0,0 +1,58 @@
+---
+theme: responsive_wide
+layout: wide
+social:
+  "og:title": "Advocate for computer science education"
+  "og:description": "Bringing together more than 50 industry, non-profit, and advocacy organizations, the Code.org Advocacy Coalition advances the development of policies to make Computer Science a fundamental part of the K-12 education system."
+  "twitter:site": "@codeorg"
+  "twitter:title": "Advocate for computer science education"
+  "twitter:description": "Learn about our local policy and access resources to support computer science in our state."
+  "twitter:url": "https://advocacy.code.org/"
+video_player: false
+---
+
+#homepage
+
+  #fullwidth{style: "position: relative"}
+    = view :header
+
+    .banner{style: "background-image: url(/images/advocacy-banner.jpg); height: 400px; background-repeat: no-repeat; background-size: cover; background-position: 50% 40%; color: white; text-align: center; xposition: absolute; top: 0; width: 100%; z-index: -5"}
```

Comment:
Can we pull out the inline CSS to a `.css` file?

Reply:
This as a WIP PR, which is why nobody was assigned, so not everything is completely done yet :)